flink-streaming-java/src/main/java/org/apache/flink/streaming/api/graph/StreamGraphGenerator.java

```diff
@@ -121,7 +120,7 @@
 
 	private boolean chaining = true;
 
-	private ScheduleMode scheduleMode = DEFAULT_SCHEDULE_MODE;
+	private ScheduleMode scheduleMode = null;
```

Comment:
I would remove the explicit `null` from here. But feel free to leave it if you prefer this.